universite-du-quebec-a-montreal.csl

```diff
@@ -276,218 +337,342 @@
           </else>
         </choose>
       </else-if>
-      <else-if type="patent" match="any">
+      <else-if type="song musical_score" match="any">
         <choose>
-          <if match="any" variable="title">
-            <group delimiter=". ">
-              <text variable="title" font-style="italic"/>
-              <text variable="number" font-style="italic"/>
-              <text variable="genre" prefix="[" suffix="]"/>
-            </group>
+          <if match="any" variable="container-title">
+            <text variable="title"/>
+            <text variable="note" prefix=" [" suffix="]"/>
           </if>
           <else>
-            <group delimiter=". ">
-              <text variable="number" font-style="italic"/>
-              <text variable="genre" prefix="[" suffix="]"/>
+            <group delimiter=" ">
+              <text variable="title" font-style="italic"/>
+              <group delimiter="; " prefix="[" suffix="]">
+                <text variable="note"/>
+                <text variable="medium"/>
+                <text variable="genre"/>
+              </group>
+              <text variable="number" prefix="(épisode " suffix=")"/>
```

Comment:
the episode label here seems highly specific and particularly odd for a musical_score type (I guess you have some sort of podcast in mind?) I don't think you're doing your users a favor by being overly specific with this.

Reply:
Yes, this is quite specific. I have added these lines based on what is described in the APA style blog for podcast: see comment at the end of the page: https://blog.apastyle.org/apastyle/2012/10/how-to-cite-a-podcast.html#comment-6a01157041f4e3970b01b8d210f59a970c
I don't expect this will be used often, but as there is an Episode Number field in Zotero for podcast (but not for audio recording or musical score), I thought it could be a good idea.